Samples~/LootLockerExamples/Scripts/PlayerFiles.cs

```diff
@@ -0,0 +1,258 @@
+
+namespace LootLocker
+{
+    public class PlayerFiles : MonoBehaviour
+    {
+        public Text informationText;
+        public Text availablePlayerFilesText;
+        public InputField playerFilesTextContent;
+        public Text playerFilesTextPlaceholder;
+
+        // Enable this to save the file on the disk as well as uploading it to LootLocker
+        public bool saveOnDisk;
+
+        public InputField fileIdUploadInput;
+        public InputField fileIdDownloadInput;
+
+        // Start is called before the first frame update
+        void Start()
+        {
+            /* 
+            * Override settings to use the Example game setup
+            */
+            LootLockerSettingsOverrider.OverrideSettings();
+
+            /* Start guest session without an identifier.
+             * LootLocker will create an identifier for the user and store it in PlayerPrefs.
+             * If you want to create a new player when testing, you can use PlayerPrefs.DeleteKey("LootLockerGuestPlayerID");
+             */
+            LootLockerSDKManager.StartGuestSession((response) =>
+            {
+                if (response.success)
+                {
+                    informationText.text = "Guest session started" + "\n";
+                    GetPlayerFiles();
+                }
+                else
+                {
+                    informationText.text = "Error" + response.Error + "\n";
+                }
+            });
+        }
+
+        public void GetPlayerFiles()
+        {
+            // Get a list of the files that the current player has
+            LootLockerSDKManager.GetAllPlayerFiles((response) =>
+            {
+                if (response.success)
+                {
+                    informationText.text += "Got player files" + "\n";
+                    availablePlayerFilesText.text = "Available files: \n";
+                    for (int i = 0; i < response.items.Length; i++)
+                    {
+                        // Write name and ID of the files to the UI
+                        availablePlayerFilesText.text += "Name:"+response.items[i].name+"\n";
+                        availablePlayerFilesText.text += "ID:" + response.items[i].id + "\n";
+                        availablePlayerFilesText.text += "Purpose:" + response.items[i].purpose + "\n";
+                        availablePlayerFilesText.text += "Size:" + response.items[i].size + "\n";
+                        availablePlayerFilesText.text += "------------------" + "\n";
+                    }
+                }
+                else
+                {
+                    informationText.text += "Error" + response.Error + "\n";
+                }
+            });
+        }
+
+        public void GetPlayerFileContent()
+        {
+            // You get the URL of the file from LootLocker
+            // You need to download the fl yourself from LootLocker.
+            // You can use the IEnumerator Download() method below
```

Comment:
I don't think we should recommend our users to actually use the example code
```suggestion
            // See the IEnumerator Download() method below for an example on how to do it
```